templates/typescript_gapic/src/$version/$service_client.ts.njk

```diff
@@ -400,29 +400,29 @@ export class {{ service.name }}Client {
       request: protosTypes{{ method.inputInterface }},
       options?: gax.CallOptions):
       Promise<[
-        protosTypes{{ method.outputInterface }},
+        protosTypes{{ method.pagingResponseType }}[],
         protosTypes{{ method.inputInterface }}|undefined, 
         {}|undefined
```

Comment:
OK I checked in gax: it calls `callback(err, resources, request, response)` or `callback(err, resources, null, response)`, so the correct types here would be:
- first parameter: `protosTypes{{ method.pagingResponseType }}[]`  (it's correct here)
- second parameter: request type or `null` (no `undefined` from what I see), so please change `undefined` to `null`
- third parameter: actual response, so it should be the response type (`IPagedExpandResponse`). I don't think it can be `undefined`.